### ISSUE TITLE
Any interest in resurrecting thesaurus?

### ISSUE THREAD

**Issue by billiegoose** [tindzk/thesaurus#1]
Hi Tim. I couldn't find a thesaurus for programmers online, so now I'm trying to put one together. I see you were working on such a thesaurus 5 years ago... I'd love to use it as a starting point!

**Comment by tindzk** [comment]
Great idea! Somehow I lost track of this project, but I would love to continue working on it. We could port the file to Markdown to make it more accessible. I also think that a different license like the Creative Commons may be a better fit.

**Comment by billiegoose** [comment]
Awesome! I [started a project](https://www.gitbook.com/book/wmhilton/programmers-thesaurus) on gitbook because I saw it used on the [vue-router documentation](http://vuejs.github.io/vue-router/en/basic.html) and was impressed, but I'll adapt to whatever platform attracts the most collaborators. (I've also been approaching potential collaborators on Twitter!) I think you're right that Creative Commons is a better fit than EUPL. I haven't used it before - mostly I publish code under a BSD/MIT/ISC - I'm thinking CC-BY-4.0 since it's the least restrictive, but I see they have several options to choose from.

**Comment by billiegoose** [comment]
I just thought of this, but we could also just use the built in Github wiki for this repo, or a README.md file

**Comment by tindzk** [comment]
Yes, I believe a `README.md` file should be fine. That way we can take pull requests more easily. I have added you as a collaborator and converted the thesaurus to Markdown. We can reuse this repository if you don't mind.

**Comment by tindzk** [comment]
Also, are you okay with [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)?

**Comment by billiegoose** [comment]
Awesome! I'll take a closer look at the README.md this evening. We might have to figure out how to spread it across multiple pages even.

If CC BY-SA is good enough for Wikipedia it's good enough for me!

**Comment by billiegoose** [comment]
Do you care if I edit the master branch directly, or should I use pull requests? How much control over the direction of the project do you want?

**Comment by tindzk** [comment]
I don't mind if you edit the master branch directly. I can invest a few hours a week. We can create issues to better coordinate changes. What ideas do you have to improve the thesaurus? I like your proposal of splitting the document into multiple pages. The only downside would be that it will be less searchable (although GitHub's search may as well work out fine).

**Comment by billiegoose** [comment]
Sorry I haven't followed up - my personal life suddenly got even busier which I didn't think was possible! I'll be back to spend time on this project again in a week or two
